packages/data-helpers/src/expand-entity/expand-entity.ts

```diff
@@ -0,0 +1,47 @@
+import { DataField, Entity, RawEntity, RawEntityData } from '../types';
+
+function expandEntity (entity: Entity): RawEntity {
+    const { data, tags } = entity;
+    const convertedData = expandEntityData(data);
+    // @ts-ignore
+    return { data: convertedData, tags };
+}
+
+function expandEntityData (data: DataField): RawEntityData {
+    const expandedEntity = Object.keys(data)
+        .reduce((expanded, key) => {
+        let converted = {};
+        switch(typeof data[key]) {
+            case 'object':
+                const isArray = data[key] instanceof Array;
+                converted = {
+                    type: isArray ? 'list' : 'object',
```

Comment:
I think this block of code (line 17-25) is a bit hard to follow, especially with the `// @ts-ignore` comments interrupting the flow. Maybe it would just been better to use an if statement?

Reply:
I have changed to if statement